components/match2/commons/match_group_input.lua

```diff
@@ -24,7 +24,7 @@ local globalVars = PageVariableNamespace()
 local MatchGroupInput = {}
 
 function MatchGroupInput.readMatchlist(bracketId, args)
```

Comment:
Changes to this function are not mentioned in the description (and don't really seem relevant)

Reply:
Removed